base/compiler/utilities.jl

```diff
@@ -212,6 +212,72 @@ function find_ssavalue_uses(e::Expr, uses::Vector{BitSet}, line::Int)
     end
 end
 
+function is_throw_call(e::Expr)
+    if e.head === :call
```

Comment:
Do we also need to handle `invoke` or is that outdated? (https://docs.julialang.org/en/v1.6-dev/devdocs/ast/#Lowered-form-1)

Reply:
`invoke` only appears post-optimization for calls to generic functions.